lib/Models/LocationMarkerUtils.ts

```diff
@@ -1,12 +1,14 @@
 import i18next from "i18next";
-import markerIcon from "./markerIcon";
 import prettifyCoordinates from "../Map/Vector/prettifyCoordinates";
 import CommonStrata from "./Definition/CommonStrata";
 import CzmlCatalogItem from "./Catalog/CatalogItems/CzmlCatalogItem";
 import Terria from "./Terria";
 import LatLonHeight from "../Core/LatLonHeight";
 import { toJS } from "mobx";
 
+//@ts-ignore
```

Comment:
I think the ts-ignore adds a lint warning, should this be `@ts-expect-error <string that explains why>`?